src/admin/site_environments.rst

```diff
@@ -0,0 +1,146 @@
+Site Environments
+=================
+
+The site environments required for running PanDA jobs.
+
+Export LSST_LOCAL_PROLOG
+------------------------
+
+For running PanDA jobs, the site only needs to export LSST_LOCAL_PROLOG.
```

Comment:
Just to clarify, this is a global export by the site?  Or is this a variable set in the user's bashrc for their interactive use and inside the bps submit yaml to tell the pilot to source it (runnerCommand)?

Reply:
This one is a global export by the site. Pilot wrapper will source it. I modified the instructions.